packages/documentation/copy/en/javascript/JSDoc Reference.md

```diff
@@ -507,34 +667,34 @@ const MathFuncs = {
 MathFuncs.add1;
 ```
 
-## `@deprecated` Comments
+## `@author`
 
-When a function, method, or property is deprecated you can let users know by marking it with a `/** @deprecated */` JSDoc comment. That information is surfaced in completion lists and as a suggestion diagnostic that editors can handle specially. In an editor like VS Code, deprecated values are typically displayed in a strike-through style ~~like this~~.
-
-```js
-// @noErrors
-/** @deprecated */
-const apiV1 = {};
-const apiV2 = {};
+You can specify the author of an item with `@author`:
 
-apiV;
-// ^|
+```ts twoslash
+/**
+ * Welcome to awesome.ts
+ * @author Ian Awesome <i.am.awesome@example.com>
```

Comment:
Maybe randos will try send a PR fixing that email because of the difference between `Ian` and `i.am` but I dig it

Reply:
In my head, Ian is 16 and very proud of publishing npm packages that other people can use (and have used!).